src/test/java/htsjdk/variant/variantcontext/AlleleUnitTest.java

```diff
@@ -277,4 +277,38 @@ public void testExtend() {
         Assert.assertEquals("ATCGA", Allele.extend(Allele.create("AT"), "CGA".getBytes()).toString());
         Assert.assertEquals("ATCGA", Allele.extend(Allele.create("ATC"), "GA".getBytes()).toString());
     }
+    @Test
+    public void testWouldBeSymbolic() {
```

Comment:
Could rewrite these tests using a testng data provider
```suggestion
    public void testWouldBeSymbolic(String allele, boolean expectedSymbolic) {
```

Reply:
Done, I didn't do all the tests in this file though, just the bottom ones..